### PR TITLE
add roadupdate oct2023 post

### PR DESCRIPTION
**Pull request by gregbunce**
and update derived dataset dates

### CODE REVIEW

**Comment by gregbunce** [comment]
A few new addresses from this road update:

- 2482 N SALVIA DR, Saratoga Springs 84043
- 1535 S WINDSONG DR, Santaquin 84655
- 1305 W 800 S, Hurricane 84737

**Comment by gregbunce** [comment]
note: i'm getting errors rendering this b/c of the dates.  i'll render this draft PR again tomorrow so they line up.

**Comment by steveoh** [comment]
Your error is 

```Liquid Exception: Could not find document "_posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md" in tag "link". Make sure the document exists and the path is correct. in data/transportation/roads-system/index.html```

**Comment by gregbunce** [comment]
thanks for rerunning the build.  it looks good to me.  let me know if i missed anything, else, please approve but not merge.  i'll merge it later today.  thanks!

File: _posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+These updates are also reflected in UGRC's [address locators]({% link data/address-geocoders-locators/index.html %}), Utah's [Next-Generation 911 GIS database]({% link data/911/index.html %}), and UGRC's [street network analysis datasets]({% link data/transportation/street-network-analysis/index.html %}).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Davis County:** Obtained roads data on 09/21/2023. Previous update was on 09/08/2023.
+- **Iron County:** Obtained roads data on 09/13/2023. Previous update was on 09/07/2022.
```
Comment (steveoh) [inline _posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md:24]: **quibble**: was iron really from 2022?

**Review by steveoh — APPROVED** [review]

File: _posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md:24]: yeah, crazy enough we still have a few counties on an annual update.  that's one of the things i'd like to improve/solve with portal and branch versioning.

File: data/transportation/roads-system/index.html [hunk 2]
```diff
@@ -136,6 +136,8 @@ <h5 id="RoadCenterlines-updates">Updates</h5>
       <p>Updates to this dataset are published every month, with approximately nine counties receiving detailed review.
       </p>
       <ul class="dotless no-padding">
+        <li><a href="{% link _posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md %}">October 06,
```
[the remark this answers is not in the document]
Comment (gregbunce) [reply data/transportation/roads-system/index.html:139]: good point.  can the github browser editor do that?  if not, i can  drop it in vscode.

[the remark this answers is not in the document]
Comment (steveoh) [reply data/transportation/roads-system/index.html:139]: i don't believe it can. vscode or possibly the codespace online

**Pull request merged**